### PR TITLE
Update IR pretty printer to accommodate FormulaValue in ResolvedObject.

### PR DESCRIPTION
**Pull request by jas-valgotar**

### CODE REVIEW

File: src/tests/Microsoft.PowerFx.Core.Tests.Shared/IRTests/IRPrettyPrinter.cs [hunk 1]
```diff
@@ -1,243 +1,248 @@
...
+        {
+            if (node.Value is NameSymbol n)
+            {
+                _sb.Append(n.Name);
+            }
+            else if (node.Value is IExternalEntity ee)
+            {
+                _sb.Append(ee.EntityName);
+            }
+            else if (node.Value is DelegationFormulaValue dfv)
```
Comment (MikeStall) [inline src/tests/Microsoft.PowerFx.Core.Tests.Shared/IRTests/IRPrettyPrinter.cs:194]: >DelegationFormulaValue [](http://example.com/codeflow?start=35&length=22)

FormulaValue?

**Comment by LucGenetier** [comment]
✅ No public API change.

**Review by MikeStall — APPROVED** [review]

File: src/tests/Microsoft.PowerFx.Core.Tests.Shared/IRTests/IRPrettyPrinter.cs [hunk 2]
```diff
@@ -190,6 +191,10 @@ public override RetVal Visit(ResolvedObjectNode node, Context context)
             {
                 _sb.Append(ee.EntityName);
             }
+            else if (node.Value is FormulaValue fv)
+            {
+                _sb.Append(fv.ToExpression());
```
Comment (MikeStall) [inline src/tests/Microsoft.PowerFx.Core.Tests.Shared/IRTests/IRPrettyPrinter.cs:196]: Do you want compact or verbose?

Comment (jas-valgotar) [reply src/tests/Microsoft.PowerFx.Core.Tests.Shared/IRTests/IRPrettyPrinter.cs:196]: I think this is fine, its just to display "smuggled" node. We can change in future if we need compact vs verbose setting.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Pull request merged**